### DISCUSSION TITLE
Last version stopped working

### DISCUSSION THREAD

**Discussion by crixodia** [community/community#11250] [Copilot Conversations]
Since I've updated to the last version, copilot stopped working. The only output I get is `Cancelled after extractPrompt`. I tried reinstalling VS Code and Copilot but it seems not working. If someone has this issue, just install `1.4688` version. 😶

**Comment by davecheney** [comment]
Hello. I’m sorry you’re having difficulties. Could you tell me what you did, what you expected, and what happened instead?

**Reply by Johanvm318** [reply]
I have the same problem, It does not work, you type something and it does not happend anything, The OUTPU appear with [INFO] [ghostText] [2022-02-28T06:20:20.774Z] Cancelled after extractPrompt

**Comment by GreatAuk** [comment]
same
